jobs/emissions.py

```diff
@@ -80,8 +80,3 @@ def main(starttime, hstart, hstop, cfg):
             source_path = os.path.join(emis_dir, emis_prefix + filename_ending)
             dest_path = os.path.join(target_dir, dest_prefix + filename_ending)
             tools.copy_file(source_path, dest_path)
-
-            # convert grid_mapping_name from string (NF90_STRING) to char
-            # (NF90_CHAR) (needed for int2lm to work)
-            if cfg.target is tools.Target.COSMO:
-                tools.string2char.main(dest_path)
```

Comment:
Did we lose some functionality, or was this not needed after all?

Reply:
I will check this. Could also be deleted by accident.

Reply:
I think it just worked well without these lines as the tests were successful locally and with jenkins. Anyways, I added this line again.